Runtime/MeshPool.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using UnityEngine;
+using UnityEngine.Pool;
+
+namespace CesiumForUnity
+{
+    public static class CesiumObjectPool
+    {
+        public static MeshPool MeshPool => meshPool;
+
+        private const int capacity = 200;
+        private static MeshPool meshPool = new MeshPool(
+                () => new Mesh(),
+                (mesh) => mesh.Clear(),
+                null,
+                (mesh) => GameObject.Destroy(mesh),
+                collectionCheck: false,
+                capacity);
+    }
+
+    public class MeshPool : ObjectPool<Mesh>
```

Comment:
Now that the C++ code isn't ever referencing this class by name, can we eliminate it entirely? Just CesiumObjectPool can simply hold an `ObjectPool<Mesh>` directly.